app/helpers/application_helper.rb

```diff
@@ -929,6 +930,8 @@ def object_types_for_flash_message(klass, record_ids)
   #
   def li_link(args)
     args[:if] = (args[:count] != 0) if args[:count]
+    args[:if] = true unless args.key(:if)
```

Comment:
@martinpovolny I believe that [Hash#key?](https://docs.ruby-lang.org/en/2.0.0/Hash.html#method-i-key-3F) is the proper method to use because [Hash#key](https://docs.ruby-lang.org/en/2.0.0/Hash.html#method-i-key) returns the value of the Hash with that key.